dot/sync/test_helpers.go

```diff
@@ -88,3 +91,43 @@ func BuildBlock(t *testing.T, instance runtime.Instance, parent *types.Header, e
 		Body:   body,
 	}
 }
+
+const (
+	defaultMinPeers     = 1
+	defaultMaxPeers     = 5
+	testTimeout         = time.Second * 5
+	defaultSlotDuration = time.Second * 6
+)
+
+func newTestChainSyncWithReadyBlocks(t *testing.T, readyBlocks *blockQueue) *chainSync {
+	header, err := types.NewHeader(common.NewHash([]byte{0}), trie.EmptyHash, trie.EmptyHash, 0,
+		types.NewDigest())
+	require.NoError(t, err)
+
+	bs := new(syncmocks.BlockState)
+	bs.On("BestBlockHeader").Return(header, nil)
+	bs.On("GetFinalisedNotifierChannel").Return(make(chan *types.FinalisationInfo, 128), nil)
+	bs.On("HasHeader", mock.AnythingOfType("common.Hash")).Return(true, nil)
```

Comment:
you should expect a particular hash, you can inject the expected `hasHeaderHash` to `newTestChainSyncWithReadyBlocks`

Reply:
Refactored `newTestChainSyncWithReadyBlocks` to remove these.